Examples/Examples/BookmarksExampleView.swift

```diff
@@ -17,40 +17,36 @@ import ArcGISToolkit
 import SwiftUI
 
 struct BookmarksExampleView: View {
+    /// Indicates whether the `Bookmarks` component is presented or not.
+    @State private var bookmarksIsPresented = false
+    
     /// The `Map` with predefined bookmarks.
     @State private var map = Map(url: URL(string: "https://www.arcgis.com/home/item.html?id=16f1b8ba37b44dc3884afc8d5f454dd2")!)!
     
-    /// Indicates if the `Bookmarks` component is shown or not.
-    /// - Remark: This allows a developer to control when the `Bookmarks` component is
-    /// shown/hidden, whether that be in a group of options or a standalone button.
-    @State var showingBookmarks = false
+    /// The selected bookmark.
+    @State private var selection: Bookmark?
     
     var body: some View {
         MapViewReader { mapViewProxy in
             MapView(map: map)
                 .toolbar {
                     ToolbarItem(placement: .primaryAction) {
```

Comment:
Having this as the primary action is what makes it bold, which I'm assuming `buttonStyle(.plain)` undoes. Better to just specify the desired placement (`topBarTrailing`).